packages/client/test/integration/Subscriber.test.ts

```diff
@@ -135,7 +135,7 @@ describe('Subscriber', () => {
             }))
     
             const receivedMessage = await nextValue(sub[Symbol.asyncIterator]())
-            expect(receivedMessage!.content).toBeString()
+            expect(receivedMessage!.content).toBeInstanceOf(Buffer)
```

Comment:
can we assert similarly we do e.g. at line 95`?`

```suggestion
            expect(receivedMessage!.content).toBeInstanceOf(Uint8Array)
```